src/ff8/vram.cpp

```diff
@@ -935,6 +944,18 @@ void ff8_wm_chara_one_upload_palette_2(int16_t *pos_and_size, uint8_t *texture_b
 	next_pal_data = nullptr;
 }
 
+void ff8_wm_update_fence_animation()
+{
+	uint8_t current_disk = uint8_t((*ff8_externals.savemap)[51]);
```

Comment:
Really minor but wouldn't `ff8_currentdisk` work in this case? it's a global var already available. Or are we looking for something else here?

Reply:
I don't know where you could use this variable, but currently it is set only for the Steam version

Reply:
Right, fair enough we don't have any other native way to detect it on both versions. Ok we may replace this in the future, not a blocker :) Thanks